lib/ansible/playbook/play_context.py

```diff
@@ -539,6 +539,17 @@ def detect_su_prompt(data):
 
                 becomecmd = '%s -u %s %s' % (exe, self.become_user, command)
 
+            elif self.become_method == 'pmrun':
+                #TODO: Add support for prompt
```

Comment:
If this is the valid prompt, then isn't it going to work when pmrun prompts for a password? My understanding of this bit of code isn't very good, but it seems like you just set the `prompt` variable and then password input is handled for you.

@bcoca is this a correct assumption?